infra/modules/bastion/main.tf

```diff
@@ -1,8 +1,8 @@
 resource "aws_eip" "elastic_ip" {
-  vpc                       = true
-  tags                      =  {
-  Name = "bastion_${var.environment}"
-                     }
+  domain = "vpc"
+  tags   =  {
+    Name = "bastion_${var.environment}"
+  }
```

Comment:
wonder if we even still need this?

Reply:
Good point - have removed and successfully deployed on AWS training account.